test/powershell/engine/Remoting/SSHRemotingAPI.Tests.ps1

```diff
@@ -2,20 +2,21 @@ Describe "SSH Remoting API Tests" -Tags "Feature" {
 
     Context "SSHConnectionInfo Class Tests" {
 
-        It "SSHConnectionInfo constructor should throw null argument exception for null UserName parameter" {
+        It "SSHConnectionInfo constructor should throw null argument exception for null UserName parameter" -Pending:$true {
 
+            # The test is pending because it is wrong or we have bug in code!
```

Comment:
Oh I see now.  Yes, this test should be removed because the code now gets the current user name if no user name is specified.  Good find.